compiler/z/codegen/OMRCodeGenerator.cpp

```diff
@@ -6947,7 +6947,24 @@ OMR::Z::CodeGenerator::findOrCreateLiteral(void *value, size_t len)
 bool
 OMR::Z::CodeGenerator::directCallRequiresTrampoline(intptrj_t targetAddress, intptrj_t sourceAddress)
    {
+#if defined(TR_TARGET_S390) && defined(TR_TARGET_64BIT) && !defined(J9ZOS390)
```

Comment:
Note: I replaced the `TR_HOST_S390` define from the OpenJ9 code with a `TR_TARGET_S390` define.